src/services/revocation.js

```diff
@@ -32,60 +32,183 @@ export class RevocationStatusServiceImpl {
    * Checks revocation status of UCAN delegations via Storage UCAN Service
    *
    * @param {Ucanto.Proof[]} proofs - Array of UCAN proofs to check
+   * @param {string} spaceDID - Space DID to validate delegation context
    * @param {import('../types/env.js').Env} env - Environment configuration
    * @returns {Promise<import('@ucanto/server').Result<boolean, import('@ucanto/server').Failure>>}
```

Comment:
Can we type this better? Should be something similar to https://github.com/storacha/ucanto/blob/d7420d5f96962a05652dd165c1fd26fa1d140f47/packages/interface/src/lib.ts#L996 i.e. return a `Revoked` error.